contrib/dokku-installer.py

```diff
@@ -100,7 +101,10 @@ def do_POST(self):
         with open('{0}/HOSTNAME'.format(dokku_root), 'w') as f:
             f.write(params['hostname'].value)
 
-        command = ['sshcommand', 'acl-add', 'dokku', 'admin']
+        user = 'admin'
+        if self.admin_user_exists():
+            user = 'web-admin'
+        command = ['sshcommand', 'acl-add', 'dokku', user]
         for key in params['keys'].value.split("\n"):
             proc = subprocess.Popen(command, stdin=subprocess.PIPE)
             proc.stdin.write(key)
```

Comment:
Okay, so the usernames should be asked though the form(web UI) or the usernames should be generated? Like `web-admin1`, `web-admin2`, etc?